packages/core/helper-plugin/lib/src/components/GenericInput/index.js

```diff
@@ -146,7 +146,9 @@ const GenericInput = ({
           hint={hint}
           required={required}
           onChange={(json) => {
-            onChange({ target: { name, value: json } });
+            // Default to null when the field is required and there is no input value
```

Comment:
About the comment, shouldn't it be when the field is not required?

Reply:
Yeah I spotted that right after pushing, and pushed a new commit https://github.com/strapi/strapi/pull/15687/commits/101b1072c23d7261af2b8b13f34b41b6e84aa51b